apps/remix-ide-e2e/src/reports/CHROME_84.0.4147.89_Mac_OS_X_sample.test.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
can we remove all the reports files or is this necessary to keep them?

Reply:
Yeah, we don't need the report files. But git ignore the directories during commit because they are empty. We will need something in it at least to get git to commit the directory.